fmt.sh

```diff
@@ -156,6 +156,7 @@ run scalafmt "${scalafmt_args[@]:-}"
 run bazel run "$buildifier_target"
 
 # Check for breaking proto changes.
+run buf mod update
```

Comment:
What does this command do? I can't find the relevant docs.

Reply:
Found it [here](https://docs.buf.build/tour/add-a-dependency/#depend-on-googleapis).

Reply:
This is needed because `completion.proto` depends on gRPC's `Status` and that's in `googleapis`.